third_party/gpus/cuda/platform.bzl

```diff
@@ -0,0 +1,39 @@
+CUDA_VERSION = "7.0"
```

Comment:
I had accidentally checked in this file after running configure. Will change CUDA_VERSION, CUDNN_VERSION and PLATFORM to be empty to make this fail loudly if a bug creeps into configure.